Say Their Names/Source/Controller/Home/HomeView.swift

```diff
@@ -0,0 +1,160 @@
+         peopleCollectionView,
+         header,
+         searchButton,
+         collections,
+         separator].forEach {
+            $0.translatesAutoresizingMaskIntoConstraints = false
+        }
+        
+        NSLayoutConstraint.activate([
+            customNavBar.leadingAnchor.constraint(equalTo: leadingAnchor),
+            customNavBar.trailingAnchor.constraint(equalTo: trailingAnchor),
+            customNavBar.topAnchor.constraint(equalTo: safeAreaLayoutGuide.topAnchor),
+            customNavBar.heightAnchor.constraint(equalToConstant: Self.CustomNavBarHeight),
+            
+            header.centerXAnchor.constraint(equalTo: customNavBar.centerXAnchor),
+            header.centerYAnchor.constraint(equalTo: customNavBar.centerYAnchor),
+            
+            searchButton.widthAnchor.constraint(equalToConstant: Self.SearchButtonSize),
+            searchButton.heightAnchor.constraint(equalToConstant: Self.SearchButtonSize),
+            searchButton.trailingAnchor.constraint(equalTo: customNavBar.trailingAnchor, constant: -Self.CustomNavBarMargin),
+            searchButton.centerYAnchor.constraint(equalTo: customNavBar.centerYAnchor),
+            
+            collections.leadingAnchor.constraint(equalTo: leadingAnchor),
+            collections.trailingAnchor.constraint(equalTo: trailingAnchor),
+            collections.topAnchor.constraint(equalTo: customNavBar.bottomAnchor),
+            collections.bottomAnchor.constraint(equalTo: bottomAnchor),
+            
+            locationCollectionView.leadingAnchor.constraint(equalTo: collections.leadingAnchor),
+            locationCollectionView.trailingAnchor.constraint(equalTo: collections.trailingAnchor),
+            locationCollectionView.topAnchor.constraint(equalTo: collections.topAnchor),
+            locationCollectionView.heightAnchor.constraint(equalToConstant: Self.PeopleCollectionViewHeight),
+            
+            separator.leadingAnchor.constraint(equalTo: collections.leadingAnchor),
+            separator.trailingAnchor.constraint(equalTo: collections.trailingAnchor),
+            separator.heightAnchor.constraint(equalToConstant: Self.SeparatorHeight),
+            separator.topAnchor.constraint(equalTo: locationCollectionView.bottomAnchor),
+            
+            peopleCollectionView.leadingAnchor.constraint(equalTo: collections.leadingAnchor),
+            peopleCollectionView.trailingAnchor.constraint(equalTo: collections.trailingAnchor),
+            peopleCollectionView.topAnchor.constraint(equalTo: separator.bottomAnchor),
+            peopleCollectionView.bottomAnchor.constraint(equalTo: collections.bottomAnchor)
+        ])
+    }
+    
+    // MARK:- Constants
+    static let CustomNavBarHeight : CGFloat = 70
+    static let PeopleCollectionViewHeight : CGFloat = 70
+    static let SearchButtonSize : CGFloat = 45
+    static let CustomNavBarMargin : CGFloat = 16
+    static let SeparatorHeight : CGFloat = 1
```

Comment:
@Kilo-Loco mentioned in another diff that we will probably need to extract such values into a separate file. Maybe we can create a `Theme.swift` or `Styles.swift`.

Reply:
I completely agree to have all the ViewCode separated into a different file.

Reply:
Agreed that this is a good idea. I've added a Trello card for this.